autonormalize/tests/test_example.py

```diff
@@ -1,12 +1,17 @@
 import featuretools as ft
+import pandas as pd
+from unittest.mock import patch
 
+import pytest
 import autonormalize as an
 
 
 def test_ft_mock_customer():
     df = ft.demo.load_mock_customer(n_customers=80, n_products=50, n_sessions=200,
                                     n_transactions=10000, return_single_table=True)
 
+    df.ww.init()
```

Comment:
Is this required? Seems like it works both with and without Woodwork initialized for me (which is the behavior I would expect), but wanted to make sure I wasn't missing something. If it's not required, I think we should remove.

Reply:
removed